supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -520,3 +520,14 @@ Do not use "BIOS" as a generic term to refer to computer firmware. Use "firmware
 *Incorrect forms*:
 
 *See also*:
+
+[discrete]
+[[bytecode]]
+==== image:images/yes.png[yes] bytecode (noun)
+*Description*: A _bytecode_ is a non-human-readable instruction set generated by a compiler. Bytecode is typically either executed by a virtual machine (VM) or recompiled into machine code. For example, Java bytecode is executed on the Java Virtual Machine (JVM).
```

Comment:
Suggest adding some text (or perhaps a comma) after "set" to clarify that it's an "instruction set" (noun), not an "instruction that is set" (verb)?

```suggestion
*Description*: A _bytecode_ is a non-human-readable instruction set that is generated by a compiler. Bytecode is typically either executed by a virtual machine (VM) or recompiled into machine code. For example, Java bytecode is executed on the Java Virtual Machine (JVM).
```

Reply:
Made @bredamc's changes.